spec/formatting.md

```diff
@@ -197,18 +197,23 @@ the following steps are taken:
 
 1. If the _expression_ includes an _operand_, resolve its value.
    If this fails, use a _fallback value_ for the _expression_.
-2. Based on the _function_ starting sigil and _name_,
+2. _Resolve the name_ of the _function_ and, based on the starting sigil,
    find the appropriate function implementation from the _function registry_.
-   If the registry does not define an implementation for this _name_,
+   If the implementation does not support the _namespace_ of the _name_
+   or the _namespace_ does not support the _name-part_ named _function_
+   or the registry does not otherwise support the _name_,
```

Comment:
```suggestion
   or the _namespace_ does not support the _name-part_ named _function_,
```

Referring to the registry here would require it to concretely exist during the formatting.

Reply:
This is the part that I *didn't* change from the original.

Reply:
Lol, you're right. So let's fix an earlier mistake.